util/reflect_test.go

```diff
@@ -844,6 +844,26 @@ var (
 	}
 )
 
+type annotatedStruct struct {
+	FieldA     *string `path:"field-a"`
+	Annotation *string `path:"@field-a" ygotAnnotation:"true"`
+}
+
+var (
```

Comment:
(nit, optional) No need for a var group for just one var.

Reply:
ACK -- moved this definition into the test where it is the only place it is used.